templates/partials/newsletter-form.html

```diff
@@ -1,51 +1,61 @@
 <form
-	class="n-newsletter-signup__form"
-	data-component="n-newsletter-signup__form"
-	data-newsletter-id="{{id}}"
-	action="
-		{{~#if userIsSubscribed~}}
-			{{unsubscribeAction}}
-		{{~else~}}
-			{{subscribeAction}}
-		{{~/if~}}
-		"
-	method="POST"
-	>
-	{{> n-myft-ui/components/csrf-token/input}}
-	<button
-		class="n-newsletter-signup__submit"
-		aria-label="
-			{{~#if userIsSubscribed~}}
-				Unsubscribe from {{name}}
-			{{~else~}}
-				One-Click Sign Up subscribe to {{name}}
-			{{~/if~}}
-		"
-		title="
-			{{~#if userIsSubscribed~}}
-				Unsubscribe from {{name}}
-			{{~else~}}
-				Subscribe to {{name}}
-			{{~/if~}}
-		"
-		data-trackable="
-			{{~#if userIsSubscribed~}}
-				newsletter-unsubscribe
-			{{~else~}}
-				newsletter-subscribe
-			{{~/if~}}
-		"
-		type="submit"
-		aria-describedby="feedback-message__newsletter-{{id}}"
-		{{#if updatingPreference}}disabled="disabled" aria-disabled="true"{{/if}}
-		>
-
-			{{#if updatingPreference}}
-				Updating<span class="o-normalise-visually-hidden">&nbsp;{{name}} subscription</span>…
-			{{else if userIsSubscribed}}
-				Unsubscribe<span class="o-normalise-visually-hidden">&nbsp;from {{name}}</span>
-			{{else}}
-				One-Click Sign Up<span class="o-normalise-visually-hidden">&nbsp;to {{name}}</span>
-			{{/if}}
-		</button>
+  class="n-newsletter-signup__form"
+  data-component="n-newsletter-signup__form"
+  data-newsletter-id="{{id}}"
+  action="
+    {{~#if userIsSubscribed~}}
+      {{unsubscribeAction}}
+    {{~else~}}
+      {{subscribeAction}}
+    {{~/if~}}
+  "
+  method="POST"
+>
+  {{> n-myft-ui/components/csrf-token/input}}
+<div
+  class="ft-concept-button ft-concept-button--follow ft-concept-button--standard"
+  data-ft-concept-button-aria-live-pressed-text="Now following {{name}} on my FT"
+  data-ft-concept-button-aria-live-unpressed-text="No longer following {{name}} on my FT"
+  data-ft-concept-button-aria-label-pressed-text="Stop following {{name}} on my FT"
+  data-ft-concept-button-aria-label-unpressed-text="Follow {{name}} on my FT"
```

Comment:
**Question / Change Request:** It looks like these are unrelated to newsletters and are for myFT follows, is there a reason they are there? If no we can remove / update them to be correct for newsletters

Reply:
I am utilizing the "Follow FT" concept button, and my understanding is that by signing up, users will also be able to view it in their "My FT" section, similar to current newsletters. This effectively acts as a newsletter sign-up. I am open to making adjustments if you suggest a better alternative.

Reply:
Unfortunately following something in myFT is not the same as signing up to a newsletter so this will need to be changed to reflect that this is a newsletter sign up

Reply:
Also a follow up question, what is using these `data` attributes?

Reply:
My bad With the data attribute I Kind of confused myself with my ft :D looking at the code now.

Regarding the data attribute, the data serves an accessibility purpose, as recommended by Origami team, to ensure users are informed about the actions that have taken place.

https://github.com/Financial-Times/origami/tree/ft-concept-button-v1.2.2/components/ft-concept-button